<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="5TS206424426L7" name="com.nbcuni.test.nbcidx.member.unlink.social.TC5332A_MemberRemove" tests="1" failures="0" timestamp="26 Sep 2014 14:49:23 GMT" time="1.638" errors="1">
  <testcase name="memberRemove" time="1.638" classname="com.nbcuni.test.nbcidx.member.unlink.social.TC5332A_MemberRemove">
    <error type="java.lang.AssertionError" message="Failed : HTTP error code : 400">
      <![CDATA[java.lang.AssertionError: Failed : HTTP error code : 400
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at com.nbcuni.test.nbcidx.member.unlink.social.TC5332A_MemberRemove.memberRemove(TC5332A_MemberRemove.java:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- memberRemove -->
</testsuite> <!-- com.nbcuni.test.nbcidx.member.unlink.social.TC5332A_MemberRemove -->
